crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -521,6 +521,7 @@ def valid_transition_states
       "applying", "discovered", "discovering", "hardware-installed",
       "hardware-installing", "hardware-updated", "hardware-updating",
       "installed", "installing", "ready", "readying", "recovering",
+      "crowbar_upgrade",
```

Comment:
Didn't you just recently say https://github.com/crowbar/barclamp-crowbar/pull/1358#issuecomment-169682244 ?

The state name is directly used for the state of the group of nodes in nodes_controller, and in relevant js parts. I do not see why it should be different here...